src/test/kotlin/com/github/avrokotlin/avro4k/schema/AvroJsonPropSchemaTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.github.avrokotlin.avro4k.schema
+
+import com.github.avrokotlin.avro4k.Avro
+import com.github.avrokotlin.avro4k.AvroJsonProp
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+import kotlinx.serialization.Serializable
+
+class AvroJsonPropSchemaTest : WordSpec() {
```

Comment:
Could you add some tests for object properties? Just to make sure that this is also correctly processed.

Reply:
Do you mean adding tests for a JSON value that is an object with properties?

Reply:
I ask because there's already the `@AvroJsonProp should support props annotation on field` test.

Reply:
> Do you mean adding tests for a JSON value that is an object with properties?

Exactly

Reply:
I simply extended that test by annotating a property using a JSON object with heterogeneous property types.